src/components/elements/CommonSnackbar.tsx

```diff
@@ -0,0 +1,23 @@
+import { Snackbar, SnackbarProps } from '@mui/material';
+
+interface CommonSnackbarProps extends SnackbarProps {}
+
+const CommonSnackbar: React.FC<CommonSnackbarProps> = ({
+  children,
+  onClose,
+  open,
+  ...props
+}) => (
+  <Snackbar
+    TransitionProps={{ appear: false }}
+    open={open}
+    onClose={onClose}
+    anchorOrigin={{ vertical: 'top', horizontal: 'center' }}
+    {...props}
+    sx={{ marginTop: '100px', ...props.sx }}
```

Comment:
Maybe it would be possible to override `top` (which seems to be what MUI Snackbar uses internally) instead of adding `marginTop`? But I couldn't get this working locally without `important`

Reply:
Oh odd, I didn't look at that, but maybe it has something to do with our sticky headers. I'm inclined to leave it as-is since it looks OK the way its appearing!